src/test/unit/views/style-guide/ut-iac-jr-leeds-daily-hearing-list.test.ts

```diff
@@ -0,0 +1,158 @@
+import sinon from 'sinon';
+import request from 'supertest';
+import { app } from '../../../../main/app';
+import { expect } from 'chai';
+import fs from 'fs';
+import path from 'path';
+import { PublicationService } from '../../../../main/service/PublicationService';
+
+const headingClass = 'govuk-heading-l';
+const bodyText = 'govuk-body';
+const govukLinkClass = 'govuk-link';
+const cell = 'govuk-table__cell';
+const tableHeader = 'govuk-table__header';
+
+describe('Upper Tribunal (Immigration and Asylum) Chamber - Judicial Review Leeds Daily Hearing List Page', () => {
+    const rawMetaData = fs.readFileSync(path.resolve(__dirname, '../../mocks/returnedArtefacts.json'), 'utf-8');
+    const rawData = fs.readFileSync(
+        path.resolve(__dirname, '../../mocks/utIacJudicialReviewLeedsDailyHearingList.json'),
+        'utf-8'
+    );
+
+    const jsonData = JSON.parse(rawData);
+
+    sinon.stub(PublicationService.prototype, 'getIndividualPublicationJson').returns(jsonData);
+    const metadataStub = sinon.stub(PublicationService.prototype, 'getIndividualPublicationMetadata');
+
+    describe('UTIAC (JR) - Leeds Daily Hearing List page', () => {
+        let htmlRes: Document;
+        const PAGE_URL = '/ut-iac-jr-leeds-daily-hearing-list?artefactId=abc';
+
+        const metaData = JSON.parse(rawMetaData)[0];
+        metaData.listType = 'UT_IAC_JR_LEEDS_DAILY_HEARING_LIST';
+
+        metadataStub.withArgs('abc').returns(metaData);
+
+        beforeAll(async () => {
+            await request(app)
+                .get(PAGE_URL)
+                .then(res => {
+                    htmlRes = new DOMParser().parseFromString(res.text, 'text/html');
+                });
+        });
+
+        it('should display header', () => {
+            const header = htmlRes.getElementsByClassName(headingClass);
+            expect(header[0].innerHTML).contains(
+                'Upper Tribunal (Immigration and Asylum) Chamber - Judicial Review: Leeds Daily Hearing List'
+            );
+        });
+
+        it('should display list date', () => {
+            const text = htmlRes.getElementsByClassName(bodyText);
+            expect(text[4].innerHTML).contains('List for 14 February 2022');
+        });
+
+        it('should display list updated date text', () => {
+            const text = htmlRes.getElementsByClassName(bodyText);
+            expect(text[5].innerHTML).contains('Last updated 20 January 2025 at 9:30am');
+        });
+
+        it('should display important information heading', () => {
+            const text = htmlRes.getElementsByClassName('govuk-details__summary-text');
+            expect(text[0].innerHTML).contains('Important information');
+        });
+
+        it('should display list update message', () => {
+            const text = htmlRes.getElementsByClassName(bodyText);
+            expect(text[6].innerHTML).contains('The following list is subject to change until 4:30pm.');
```

Comment:
It will work but can you add full open justice wording to make it consistent with other lists.